mozfun/norm/examples/fenix_app_info.sql

```diff
@@ -1,9 +1,34 @@
+-- Example of a query over all Fenix builds advertised as "Firefox Beta"
```

Comment:
This example should make it clear that there's still a lot of complexity involved in unioning tables together to get complete answers per channel.